training_data_collector/main/chessdotcom/labelgame.py

```diff
@@ -0,0 +1,55 @@
+from selenium.webdriver.common.by import By
+import cv2
+
+from chessdotcom import actions
+from darknetconvert import convert_co_ords_to_darknet
+from flags import flags
+from image_enrichment import enrich_image
+
+def label_game(driver):
+    gameName = flags['game_id']
+    moveCount = actions.get_move_count(driver)
+    board_size = flags['min_board_size']
+
+    for x in range(1, moveCount + 1):
+        if board_size != flags['max_board_size']:
+            board_size += flags['board_size_increment']
+            # actions.change_board_size(driver, board_size)
```

Comment:
what da comment doin? can it be removed?

Reply:
Removed